docs/content/preview/explore/query-1-performance/yb-pg-stat-get-queries.md

```diff
@@ -0,0 +1,272 @@
+- Terminated by SIGKILL - the query was killed by the system's out of memory killer because the node is running out of memory.
+
+The function takes a single argument, the Object identifier (OID) of the database of interest. If no OID is provided, it returns terminated queries for all databases.
+
+{{% explore-setup-single %}}
+
+## Supported fields
+
+At a `ysqlsh` prompt, run the following meta-commands to return the fields supported by yb_pg_stat_get_queries:
+
+```sql
+yugabyte=# \x
+yugabyte=# \df yb_pg_stat_get_queries
+```
+
+```output
+List of functions
+-[ RECORD 1 ]-------+-----------------------------------------------------
+Schema              | pg_catalog
+Name                | yb_pg_stat_get_queries
+Result data type    | SETOF record
+Argument data types | db_oid oid, OUT db_oid oid, OUT backend_pid integer, 
+                    | OUT query_text text, OUT termination_reason text, 
+                    | OUT query_start timestamp with time zone, 
+                    | OUT query_end timestamp with time zone
+Type                | func
+```
+
+The following table describes the arguments and their values:
+
+| Field | Type | Description |
+| :---- | :--- | :---------- |
+| db_oid | OID | OID of the database to which the backend was connected when the query was terminated. |
+| backend_pid | Integer | Backend process ID. |
+| query_text | Text | The query that was executed, up to a maximum of 256 characters. |
+| termination_reason | Text | An explanation of why the query was terminated. One of:<br/>Terminated by SIGKILL<br/>Terminated by SIGSEGV<br/>temporary file size exceeds temp_file_limit (xxx kB)
+| query_start | Timestampz | Time at which the query started. |
+| query_end | Timestampz | Time at which the query was terminated. |
+
+## Examples
+
+### PostgreSQL crash
+
+To simulate a crash in the PostgreSQL process, send a SIGSEGV signal to the backend process.
+
+In a ysqlsh session, get the backend pid.
+
+```sql
+yugabyte=# SELECT pg_backend_pid();
+```
+
+```output
+ pg_backend_pid
+----------------
+           4650
+(1 row)
+```
+
+In the same session, start a long-running query, so that you have time to send a signal while the query is running.
+
+```sql
+yugabyte=# SELECT * FROM generate_series(1, 123456789);
+```
+
+In another shell, send the terminating signal to the backend process.
+
+```sh
+$ kill -SIGSEGV 4650 # the pid of the backend process
+```
+
+Verify that the query is listed as a terminated query.
+
+```sql
+yugabyte=# SELECT backend_pid, query_text, termination_reason FROM yb_pg_stat_get_queries(NULL);
+```
+
+```output
+ backend_pid |                  query_text                  |  termination_reason
+-------------+----------------------------------------------+-----------------------
+        4650 | SELECT * FROM generate_series(1, 123456789); | Terminated by SIGSEGV
+(1 row)
+```
+
+### Exceed the temporary file limit
+
+To simulate a query termination, set `temp_file_limit` to 0kB.
```

Comment:
To simulate a query termination, set `temp_file_limit` to 0KB:

(I think we usually use KB instead of kB)